controllers/watcher_controller.go

```diff
@@ -189,6 +193,19 @@ func (r *WatcherReconciler) Reconcile(ctx context.Context, req ctrl.Request) (re
 	_ = op
 	// end of TransportURL creation
 
+	// Create Keystone Service creation. The endpoint will be created by WatcherAPI
+	_, err = r.ensureKeystoneSvc(ctx, helper, instance, serviceLabels)
+
+	if err != nil {
+		return ctrl.Result{}, err
```

Comment:
I think we should set `KeystoneServiceReadyCondition` to False here to mark that there was a problem creating the keystone service

Reply:
right, I think i will do it in ensureKeystoneSvc so that we keep the main reconcile cleaner.